### PR TITLE
Introduce component library

### PR DESCRIPTION
**Pull request by robiot**
image
image
image
image

### CODE REVIEW

**Comment by robiot** [comment]
This "component library" is completely unused in this PR. 
A new PR will be created for migrating to the component library.

**Comment by robiot** [comment]
image
image

With these components, the design will stay consistent across the whole page.

File: apps/app/src/components/Modal.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const Modal: FC<ModalComponent> = ({
                         className
                     )}
                 >
-                    <div className="relative bg-white rounded-lg shadow dark:bg-neutral-800 border-white border">
+                    <div className="relative bg-white rounded-lg shadow dark:bg-neutral-900 border-neutral-300 dark:border-neutral-800 border">
```
Comment (lucemans) [inline apps/app/src/components/Modal.tsx:30]: 😏

File: apps/app/src/pages/login/StepOne.tsx [hunk 2]
```diff
@@ -11,9 +11,9 @@ export const LoginStepOne: FC = () => {
                 <img src={logo} alt="" className="w-24" />
                 <h1 className="text-2xl">EDGESERVER.io</h1>
             </div>
-            <div className="w-full h-64 bg-neutral-200 dark:bg-neutral-700 rounded-lg p-6">
+            {/* <div className="w-full h-64 bg-neutral-200 dark:bg-neutral-700 rounded-lg p-6">
                 Hi there this box is placeholder
-            </div>
+            </div> */}
```
Comment (lucemans) [inline apps/app/src/pages/login/StepOne.tsx:16]: Thank you

File: apps/app/tailwind.config.js [hunk 3]
```diff
@@ -14,11 +14,11 @@ module.exports = {
...
-                blue: {
-                    600: '#0A84FF',
-                    700: 'rgb(0 107 214)',
-                    800: '#102841',
-                },
+                // blue: {
+                //     600: '#066bff',
+                //     700: 'rgb(0 107 214)',
+                //     // 800: '#102841',
+                // },
```
Comment (lucemans) [inline apps/app/tailwind.config.js:21]: ?

**Comment by lucemans** [comment]
Preferrably use `Link` from `react-router` so that the SPA routing functionality works!

File: packages/components/vite.config.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            name: '@edgelabs/ui',
+            formats: ['es', 'umd'],
+            fileName: (format) => `index.${format}.js`,
+        },
+        // rollupOptions: {
+        //     external: [...Object.keys(packageJson.peerDependencies)],
+        // },
+        // rollupOptions: {
+        //     plugins: [nodePolyfills()],
+        // },
```
Comment (lucemans) [inline packages/components/vite.config.ts:33]: ?
```suggestion
```

**Review by lucemans — CHANGES_REQUESTED** [review]

**Review by lucemans — APPROVED** [review]
LGTM

**Pull request merged**